### ISSUE TITLE
First example in the docs broken on OSX

### ISSUE THREAD

**Issue by santiagobasulto** [pybee/toga#61]
Using the example in this tutorial: https://toga.readthedocs.io/en/latest/tutorial/tutorial-0.html

I get the following error:

```
Traceback (most recent call last):
  File "toga_test.py", line 25, in <module>
    app.main_loop()
  File "/Users/santiagobasulto/.virtualenvs/toga/lib/python2.7/site-packages/toga_cocoa/app.py", line 103, in main_loop
    self._startup()
  File "/Users/santiagobasulto/.virtualenvs/toga/lib/python2.7/site-packages/toga_cocoa/app.py", line 92, in _startup
    self.startup()
  File "/Users/santiagobasulto/.virtualenvs/toga/lib/python2.7/site-packages/toga_cocoa/app.py", line 99, in startup
    self.main_window.content = self._startup_method(self)
  File "toga_test.py", line 15, in build
    button.style.set(margin=50)
AttributeError: 'Button' object has no attribute 'style'
```

**Comment by elstob** [comment]
It appears PyBee pushed a new CSS based layout system 7 days ago but the pip package hasn't been updated to reflect this yet. You can manually install the latest version directly from Github using `pip install git+https://github.com/pybee/toga.git` but I don't know if you'll also have to update other packages such as toga-cocoa to get it to work.

**Comment by freakboy3742** [comment]
@santiagobasulto @elstob's comment is correct - the tutorial isn't updated for the new code. I apologise for the disconnect - I'm in the process of cleaning things up.

**Comment by Neophen** [comment]
Trying to run the first example of toga. Ended up shaving yaks :D A little bit of irony :)
What to use instead of style? or is this abandoned now?

**Comment by Neophen** [comment]
Actually i'm starting out developing in python, and really see that toga would be a great thing, is there anyway I could contribute, consider that i'm quite a newbie and never contributed to a project :) Let me know if I can help out somehow.

**Comment by Ocupe** [comment]
The problem still persist. I don't mean to push, but is there a way to fix it by now? :)

**Comment by freakboy3742** [comment]
This has now been resolved by the 0.2.0 release.
